### ISSUE TITLE
[TEST-FAILURE] ClientReplicatedMapListenerTest.testEntryUpdated

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#6545]
```
java.lang.AssertionError: expected:<1> but was:<2>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.client.replicatedmap.ClientReplicatedMapListenerTest$2.run(ClientReplicatedMapListenerTest.java:79)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:827)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:841)
	at com.hazelcast.client.replicatedmap.ClientReplicatedMapListenerTest.testEntryUpdated(ClientReplicatedMapListenerTest.java:76)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-OpenJDK7/com.hazelcast$hazelcast-client/696/testReport/junit/com.hazelcast.client.replicatedmap/ClientReplicatedMapListenerTest/testEntryUpdated/

**Comment by Donnerbart** [comment]
**Error Message**
expected:<1> but was:<2>

**Stacktrace**
```
java.lang.AssertionError: expected:<1> but was:<2>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.client.replicatedmap.ClientReplicatedMapListenerTest$2.run(ClientReplicatedMapListenerTest.java:75)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:825)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:839)
	at com.hazelcast.client.replicatedmap.ClientReplicatedMapListenerTest.testEntryUpdated(ClientReplicatedMapListenerTest.java:72)
```

**Standard Output**
```
Finished Running Test: ClientReplicatedMapListenerTest.testListenWithPredicate in 5.331 seconds.
18:07:36,287  INFO [LifecycleService] testEntryUpdated - HazelcastClient[hz.client_1032_dev][3.6-RC1-SNAPSHOT] is SHUTTING_DOWN
18:07:36,287  INFO [LifecycleService] testEntryUpdated - HazelcastClient[hz.client_1032_dev][3.6-RC1-SNAPSHOT] is SHUTDOWN
18:07:36,287  INFO [LifecycleService] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Address[127.0.0.1]:6138 is SHUTTING_DOWN
18:07:36,287  WARN [Node] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Terminating forcefully...
18:07:36,288  INFO [Node] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Shutting down multicast service...
18:07:36,288  INFO [Node] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Shutting down connection manager...
18:07:36,288  INFO [Node] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Shutting down node engine...
18:07:36,293  INFO [NodeExtension] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Destroying node NodeExtension.
18:07:36,293  INFO [Node] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Hazelcast Shutdown is completed in 6 ms.
18:07:36,293  INFO [LifecycleService] testEntryUpdated - [127.0.0.1]:6138 [dev] [3.6-RC1-SNAPSHOT] Address[127.0.0.1]:6138 is SHUTDOWN
```
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-pr-builder/com.hazelcast$hazelcast-client-legacy/16814/testReport/junit/com.hazelcast.client.replicatedmap/ClientReplicatedMapListenerTest/testEntryUpdated/

**Comment by sancar** [comment]
fixed by #6728
